include/extractor/edge_based_graph_factory.hpp

```diff
@@ -46,28 +47,34 @@ namespace extractor
 namespace lookup
 {
 // Set to 1 byte alignment
+PACK(
 struct SegmentHeaderBlock
 {
     std::uint32_t num_osm_nodes;
     OSMNodeID previous_osm_node_id;
-} __attribute ((packed));
+}
+);
 static_assert(sizeof(SegmentHeaderBlock) == 12, "SegmentHeaderBlock is not packed correctly");
 
+PACK(
 struct SegmentBlock
 {
     OSMNodeID this_osm_node_id;
     double segment_length;
     std::int32_t segment_weight;
-} __attribute ((packed));
+}
+);
 static_assert(sizeof(SegmentBlock) == 20, "SegmentBlock is not packed correctly");
 
+PACK(
 struct PenaltyBlock
 {
     std::uint32_t fixed_penalty;
     OSMNodeID from_id;
     OSMNodeID via_id;
     OSMNodeID to_id;
-} __attribute ((packed));
+}
```

Comment:
Thanks @oxidase , I'll update it soon.  That was the effect I want to achieve, but it bites back on my face :(